docs/components/camera/transform.md

```diff
@@ -181,14 +184,16 @@ The Crop transform crops takes an image and crops it to a rectangular area speci
 The Depth Edges transform creates a canny edge detector to detect edges on an input depth map.
 
 ```json {class="line-numbers linkable-line-numbers"}
-{
-  "type": "depth_edges",
-  "attributes": {
-    "high_threshold_pct": <float>,
-    "low_threshold_pct": <float>,
-    "blur_radius_px": <float>
+"pipeline": [
```

Comment:
I think this still would work if you posted it into the advanced field. So it's still confusing. I think we need to add the entire json component config object

Reply:
adding source camera-- I think this is what you mean right? making it look like the attributes do in the advanced view when configured fully